docs/management/ingest-pipelines/ingest-pipelines.asciidoc

```diff
@@ -62,47 +62,63 @@ You also want to know where the request is coming from.
 
 . In *Ingest Node Pipelines*, click *Create a pipeline*.
 . Provide a name and description for the pipeline.
-. Define the processors:
+. Click *Add a processor* to define your first processor.
+. Select the *Grok* processor type from the dropdown menu.
++
+Set the field input to `message` and provide the following pattern:
 +
 [source,js]
 ----------------------------------
-[
-    {
-      "grok": {
-        "field": "message",
-        "patterns": ["%{IPORHOST:clientip} %{USER:ident} %{USER:auth} \\[%{HTTPDATE:timestamp}\\] \"%{WORD:verb} %{DATA:request} HTTP/%{NUMBER:httpversion}\" %{NUMBER:response:int} (?:-|%{NUMBER:bytes:int}) %{QS:referrer} %{QS:agent}"]
-      }
-    },
-    {
-      "date": {
-        "field": "timestamp",
-        "formats": [ "dd/MMM/YYYY:HH:mm:ss Z" ]
-      }
-    },
-    {
-      "geoip": {
-        "field": "clientip"
-      }
-    },
-    {
-      "user_agent": {
-        "field": "agent"
-      }
-    }
-  ]
+%{IPORHOST:clientip} %{USER:ident} %{USER:auth} \[%{HTTPDATE:timestamp}\] "%{WORD:verb} %{DATA:request} HTTP/%{NUMBER:httpversion}" %{NUMBER:response:int} (?:-|%{NUMBER:bytes:int}) %{QS:referrer} %{QS:agent}
 ----------------------------------
 +
-This code defines four {ref}/ingest-processors.html[processors] that run sequentially:
+Once complete, click *Update*.
+. Add 3 more processors with the following data:
++
+*Date*
```

Comment:
There might be a better way to display this data 😄

Reply:
Yeah, especially the way that the table is rendered. I think I'd split up the steps differently, instead. Something like:

```
. Add a grok processor to parse the log message:

.. Click *Add a processor* and select the *Grok* processor type.
.. Set the field input to `message` and enter the following grok pattern.
.. Click *Update* to save the processor.

. Add processors to map the Date, GeopIP, and User agent fields:

.. Select the ***Field*** processor type.
.. Map the appropriate parsed value to each field:
+
--
* Date: `timestamp`
* GeoIp: `clientip`
* User agent: `agent`

For the Date field, you also need to specify the date format you want to use.
--
```

Reply:
Thanks! I like this a lot better. I followed this pattern, but changed the copy in a few places where I felt it wasn't 100% clear.